README.md

```diff
@@ -1,5 +1,5 @@
 [![CircleCI](https://circleci.com/gh/julianguyen/ifme/tree/master.svg?style=svg)](https://circleci.com/gh/julianguyen/ifme/tree/master)
```

Comment:
For the headings, Contributors, Backers, and Sponsors let's add that to the Table of Contents